haverhill_311_function/modules/db.py

```diff
@@ -53,6 +53,82 @@ class QAlertRequest(Base):
     )
 
 
+class QAlertAudit(Base):
+    """Sqlalchemy orm model for the qalert audit table"""
+    __tablename__ = 'qalert_audits'
+    id = Column(Integer, primary_key=True)
+    create_date = Column(Text)
+
+
+class QAlertAuditDB:
```

Comment:
lot of duplication between this class and the QAlertDB class, id recommend abstracting a DB class, and passing a schema class as param but up to u.

Reply:
Yupyup, will consider it for later! Just wanted to get a working version first